pkgs/data/icons/flat-remix-icon-theme/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, gtk3 }:
+
+stdenv.mkDerivation rec {
+  name = "${package-name}-${version}";
+  package-name = "flat-remix-icon-theme";
```

Comment:
Use `pname` instead, then you don't need to set the name either and you can remove the `rec` too.

Reply:
I made the changes. Thanks